package.json

```diff
@@ -39,7 +39,7 @@
   },
   "homepage": "https://redux-saga.js.org/",
   "dependencies": {
-    "redux": "^3.5.1"
+    "redux": ">=0.10 <5"
```

Comment:
is it compatible with version < `3.5.1`? I don't know why it was selected before

Reply:
Previously I've just installed redux and it got me latest version at that time in the range. Later I've backtracked in which version `compose` got introduced to redux and it was 0.10

Reply:
Curiously, on the tests I've made the installed redux version was 3.5.2, because react and some other peer dependencies seem to have not been updated to support redux 4.0 yet (like redux-devtools, for example).

But I still think that external users should not suffer from that problem and may use redux 4.0 easily with this change.